environments/workspaces-as-code.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Workspaces As Code"
+description: "Learn how to describe environment configuration as code."
+state: beta
+---
+
+Workspaces as Code (WAC) brings Infrastructure As Code to environment
+configuration. Users can use this feature to manage environment config in the
+same repo as their source code; simply add a `.coder/coder.yaml` to get started.
```

Comment:
Workspaces as Code (WAC) brings the Infrastructure as Code paradigm to Coder environments. [Workspace Templates[(<-- add link -->) are declarative YAML files that describe how to configure environments and their supporting infrastructure. To get started, add a `.coder/coder.yaml` file to your repository.